pcdsdevices/signal.py

```diff
@@ -335,21 +341,56 @@ class UnitConversionDerivedSignal(DerivedSignal):
         Keyword arguments are passed to the superclass.
     """
 
+    derived_units: str
+    original_units: str
```

Comment:
I've never seen this before. Is this how you type hint a class attribute?

Reply:
Yes, it's the new-style class instance attribute type hint annotation style.

Apologies for inserting these here in the first place; my side-project coding style has been leaking out of its own context recently.

Reply:
To be clear, I don't mind removing these at your discretion as these don't really fit with the code style here.